src/coreComponents/dataRepository/DataContext.hpp

```diff
@@ -0,0 +1,205 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file DataContext.hpp
+ */
+
+#ifndef GEOS_DATAREPOSITORY_SOURCECONTEXT_HPP_
```

Comment:
To update to GEOS_DATAREPOSITORY_DATACONTEXT_HPP_
 as well as the endif comment